youtube_dl/extractor/bostonglobe.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+from ..utils import (
+    extract_attributes,
+)
+
+
+class BostonGlobeIE(InfoExtractor):
+    _VALID_URL = r'https?(?i)://(?:www\.)?bostonglobe\.com/.*/(?P<id>[^/]+)/\w+(?:\.html)?'
```

Comment:
Done. Also with a re-sorting of `common.py`'s import clause. Although I am confused:

> Flags like (?i) must be in the start of a regular expression since Python 3.6. See http://bugs.python.org/issue22493

Why does this not show up in the Travis output for Python 3.6?: https://travis-ci.org/rg3/youtube-dl/jobs/208950149
It sure would have been helpful to know.

Thanks again.